docs/index.rst

```diff
@@ -288,37 +288,60 @@ reduces performance overhead.
 Distributed Tracing
 ~~~~~~~~~~~~~~~~~~~
 
-To trace requests across hosts, the spans on the secondary hosts must be linked together by setting `trace_id` and `parent_id`::
+To trace requests across hosts, the spans on the secondary hosts must be linked together by setting `trace_id`, `parent_id` and `sampling_priority`.
 
-    def trace_request_on_secondary_host(parent_trace_id, parent_span_id):
-        with tracer.trace("child_span") as span:
-            span.parent_id = parent_span_id
-            span.trace_id = parent_trace_id
+`ddtrace` already provides default propagators but you can also implement your own.
 
+HTTP
+~~~~
 
-Users can pass along the parent_trace_id and parent_span_id via whatever method best matches the RPC framework. For example, with HTTP headers (Using Python Flask)::
+The `HTTPPropagator` is already automatically used in our `aiohttp` integration. For the others, you can use
+it manually.
```

Comment:
I guess we want to do that automatically for all of them.... no? I think probably only `requests` should be done manually because that way we remove the need to change our current implementation to have per-integration configs (`distributed_tracing=True`).

Or this is just something to make this PR consistent with the current implementation?

Reply:
Yes totally, I'll leave that to another task/PR to bring it to all our existing instrumentations + update the doc.